### PR TITLE
Issue #173 follow-up - Move Extensions text into a separate subsection

### PR DESCRIPTION
**Pull request by robstradling**

### CODE REVIEW

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -690,6 +688,30 @@ Future versions of this protocol may reuse `VersionedTransType` values defined
...
+        reserved(65535)
+    } ExtensionType;
+
+    struct {
+        ExtensionType extension_type;
+        opaque extension_data<0..2^16-1>;
+    } Extension;
+~~~~~~~~~~~
+
+The `Extension` structure provides a generic extensibility for log entries,
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Nit: The term 'log entries' here is a bit vague: SCTs and STHs are not log entries. They are artifacts produced by the log. Maybe it'd be better to replace it with 'log artifacts' ?

File: draft-ietf-trans-rfc6962-bis.md [hunk 2]
```diff
@@ -1914,10 +1919,10 @@ SCTs and other `TransItem` structures.
 The appointed Expert should review the public specification to ensure that it is
 detailed enough to ensure implementation interoperability.
 
-## Extensions    {#extension_types}
+## Extension Types    {#extension_types}
 
-IANA is asked to establish a registry of SCT extensions, named "CT Extension
-Types for SCT", that initially consists of:
+IANA is asked to establish a registry of `ExtensionType` values, named "CT
+Extension Types", that initially consists of:
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Consider renaming the registry to CT log artifact extension types?

File: draft-ietf-trans-rfc6962-bis.md (hunk 1 above)
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: It doesn't say that SCTs and STHs are log entries.  "log entries" is the first of three items in a list; the second item is SCTs, and the third item is STHs.

However, to make it clear, I'll do "s/entries,/artifacts, including/".

File: draft-ietf-trans-rfc6962-bis.md (hunk 2 above)
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: OK.

File: draft-ietf-trans-rfc6962-bis.md [hunk 3]
```diff
@@ -1914,10 +1919,10 @@ SCTs and other `TransItem` structures.
 The appointed Expert should review the public specification to ensure that it is
 detailed enough to ensure implementation interoperability.
 
-## Extensions    {#extension_types}
+## Extension Types    {#extension_types}
 
-IANA is asked to establish a registry of SCT extensions, named "CT Extension
-Types for SCT", that initially consists of:
+IANA is asked to establish a registry of `ExtensionType` values, named "CT Log
+Artifact ExtensionTypes", that initially consists of:
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Nit: "Extension Types" (two words)

Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: "ExtensionType" is one word though, and it matches ```IANA is asked to establish a registry of `VersionedTransType` values, named "CT VersionedTransTypes```.

i.e., We didn't write ```named "CT VersionedTrans Types"```

Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: Actually, let's: 1) call it "CT Log Artifact Extensions", and 2) rename the "Value" column to "ExtensionType", and 3) rename the "Extension" column to "Status".

**Comment by robstradling** [comment]
Merged at d4fb5ce5c32e27f83eabee818a939f63a5cad5ca

**Pull request closed without merging**